test/e2e/webdriver/driver.js

```diff
@@ -63,12 +63,6 @@ function wrapElementWithAPI(element, driver) {
     }
   };
 
-  element.nestedFindElement = async (rawLocator) => {
```

Comment:
I think this is somewhat inconsistent with the other webdriver methods for finding elements [findElement(s), findClickableElement(s), findVisibleElement, etc.]